WordPress/src/main/java/org/wordpress/android/ui/stories/media/StoryMediaSaveUploadBridge.kt

```diff
@@ -172,7 +172,7 @@ class StoryMediaSaveUploadBridge @Inject constructor(
                             // ids with the new MediaModel local ids are created so, broadcasting the event.
                             if (isEditMode) {
                                 // finally send the event that this frameId has changed
-                                EventBus.getDefault().post(
+                                EventBus.getDefault().postSticky(
```

Comment:
Suggesting this for unison with the other EventBus calls 😜 
```suggestion
                                eventBusWrapper.postSticky(
```